### ISSUE TITLE
Can't locate DiffHighlight.pm in @INC (Windows 10)

### ISSUE THREAD

**Issue by jedi-tester** [so-fancy/diff-so-fancy#265]
Hi,

I'm having troubles installing this plugin on win10.

I'm using Git for Windows (portable version) and PowerShell.
Copied diff-so-fancy folder to PortableGit folder and executed from PowerShell following command: `git config --global core.pager "'C:\Portable\PortableGit\diff-so-fancy\diff-so-fancy' | less --tabs=4 -RFX"`

Can you please add a clearer instruction for installation on Win10.

Thank you.

**Comment by scottchiefbaker** [comment]
Where did you install DiffHighlight.pm?

**Comment by jedi-tester** [comment]
@scottchiefbaker apparently I haven't. Readme says to copy script, that's what I did initially.
How do I install it? Is it possible to build from source or installation is only available via npm?

**Comment by scottchiefbaker** [comment]
What version are you running? If you're using 1.1.0 there should be no need for an external `DiffHighligh.pm`

**Comment by jedi-tester** [comment]
Sorry for late reply @scottchiefbaker , got swamped at work. I'm using 1.1.1
Should I try 1.1.0 instead?

**Comment by scottchiefbaker** [comment]
If you cloned the repo you will need to have `lib/DiffHighlight.pm` relative to wherever you have `diff-so-fancy` installed. Alternately you can use the [fatpack version](https://raw.githubusercontent.com/so-fancy/diff-so-fancy/master/third_party/build_fatpack/diff-so-fancy) which is a single script.

**Comment by jedi-tester** [comment]
That's the point, I have lib/DiffHighlight.pm in the installed folder, but when running it I'm getting the above mentioned error.
Let me try again tomorrow and I'll post my results.

**Comment by jedi-tester** [comment]
Using fatpack version seems to be working fine. 
Could not make it working when cloning the repo but I suppose the issue can be closed.

**Comment by scottchiefbaker** [comment]
@jedi-tester looks like @stanislavromanov has a fix for this in #267. Can you test it on Windows? It works for me on Linux.

**Comment by jedi-tester** [comment]
@scottchiefbaker yes, will test this tomorrow and post the results. Thanks

**Comment by jedi-tester** [comment]
Seems to be fixed. Thanks.

**Comment by scottchiefbaker** [comment]
Closed via #267
